packages/desktop/src/rtl-tests/channel.test.tsx

```diff
@@ -775,6 +796,19 @@ describe('Channel', () => {
         }
       })
 
+    initialState.dispatch(
```

Comment:
It's minor but placing store transformations next to the objects mocking section makes the whole test better readable